aws-lambda/jaeger-ruby/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenTracing in AWS Lambda Ruby 2.5
+
+This is an example Lambda function that shows how to use the Ruby Jaeger client to send spans to SignalFx.
```

Comment:
I thought the whole point here was to use our Lambda Ruby wrapper

Reply:
Will update it to use the wrapper